js/components/tab.js

```diff
@@ -18,6 +18,8 @@ const dnd = require('../dnd')
 const windowStore = require('../stores/windowStore')
```

Comment:
If you move the new tabIcon component to the new location (`app/renderer/components`), you may consider moving this too :smile: If moved, you'll have to update the paths for the requires here and then also anywhere that includes this (ex: tabs, pinnedtabs)